src/Service/Accounts/Resource/FirebaseTokens.php

```diff
@@ -25,22 +25,20 @@
 class FirebaseTokens extends Resource
 {
     /**
-     * @var string
+     * @var FirebaseToken
      */
-    public $userToken;
+    public $shop;
 
     /**
-     * @var string
+     * @var FirebaseToken
      */
-    public $userRefreshToken;
+    public $pointOfContact;
 
-    /**
-     * @var string
-     */
-    public $shopToken;
+    public function __construct($values = [])
+    {
+        $values['shop'] = new FirebaseToken($values['shop']);
+        $values['pointOfContact'] = new FirebaseToken($values['pointOfContact']);
```

Comment:
```suggestion
        $this->castChildResource($values, FirebaseToken::class, [
            'shopToken',
        ]);
        $this->castChildResource($values, FirebaseToken::class, [
            'pointOfContactToken',
        ]);
```
Also I think it's a bit weird without the `Token` prefix on the members.